client/src/components/avatar/EntityAvatarEditModal.js

```diff
@@ -0,0 +1,133 @@
+import { preventZoom } from "advanced-cropper/extensions/prevent-zoom"
+import AttachmentCard from "components/Attachment/AttachmentCard"
+import PropTypes from "prop-types"
+import React, { useEffect, useRef, useState } from "react"
+import { Cropper, RectangleStencil } from "react-advanced-cropper"
+import { Button, Modal } from "react-bootstrap"
+import Settings from "settings"
+
+const EntityAvatarEditModal = ({ title, avatar, images, onAvatarUpdate }) => {
+  const avatarCroppeableMimeTypes = Settings.fields.attachment.mimeTypes
```

Comment:
Sinced we're already in the context of an avatar, maybe rename this to something like `croppableMimeTypes`?

Reply:
Sure